src/ranged.cpp

```diff
@@ -341,29 +424,52 @@ dealt_projectile_attack Creature::projectile_attack( const projectile &proj_arg,
     return attack;
 }
 
+static double erfinv( double x )
```

Comment:
You need to unit test this somehow. Checking the output is commutative with inverse function from the C++ standard library might be one option?